assets/js/admin.js

```diff
@@ -207,25 +207,6 @@ const Admin = {
 
   setup_xeditable(subject) {
     Admin.log('[core|setup_xeditable] configure xeditable on', subject);
```

Comment:
Why not completely remove the function alltogether? To my knowledge this function is not called anywhere else outside of the `admin.js` file.

Reply:
Fixed